src/network/modules/tokens/tokens.ts

```diff
@@ -15,7 +17,9 @@ import {
 export const Tokens: TokensModule = {
   _namespace_: "tokens",
   async info(param: TokensInfoParam): Promise<TokenInfo> {
-    const data = new Map([[0, param.address]])
+    const data = new Map([
+      [0, tag(10000, Address.fromString(param.address).toBuffer())],
```

Comment:
It might be a very dumb question because I don't do FE, but is there any way to avoid manually tagging addresses every time?

Reply:
No, you're right. I was surprised that we don't already have a utility function for this conversion. We have one for `getAddressFromTaggedIdentity`. I can definitely add one but I'm still trying to understand if there's a hidden logic to how we've set up ManyJS or if it has just evolved organically and could use some refactoring.